src/main/java/seedu/address/model/course/Course.java

```diff
@@ -62,6 +55,27 @@ public int getCourseSize() {
         return this.students.size();
     }
 
+    /**
+     * Returns the number of students tagged with GOOD StudentRank.
+     */
+    public long getGoodTagCount() {
```

Comment:
for this and the other similar methods, any particular reason why long instead of int? just curious as the extra range of long (as opposed to int) may not be necessary for this application